app/master/build.py

```diff
@@ -523,6 +527,22 @@ def _delete_temporary_build_artifact_files(self):
         end_time = time.time() - start_time
         self._logger.info('Completed deleting artifact files for {}, took {:.1f} seconds.', self._build_id, end_time)
 
+    def _update_prometheus_metrics(self):
+        serialized_time_val, estimation_error_val = 0, 0
+        for subjob in self._all_subjobs_by_id.values():
+            for atom in subjob.atoms:
+                if atom.actual_time is None:
+                    self._logger.warning('Could not load atom.actual_time for atom {} build {}', atom.command_string, self._build_id)
+                    return
+                if atom.expected_time is None:
+                    self._logger.warning('Could not load atom.expected_time for atom {} build {}', atom.command_string, self._build_id)
```

Comment:
I think this warning will create a ton of noise the first time a job is run since there are no expected times in that case. How do you feel about removing this warning as well? I feel like it would be reasonable for `atom.expected_time` to be set to `None` in the case of a new atom, even if that's not the current behavior.

Also, does it make sense to return if expected time is none? It seems like even if we don't have any expected times set for a build that we could still record the serialized build time.